### DISCUSSION TITLE
Update list of Tagalog stop words

### DISCUSSION THREAD

**Discussion by AFAgarap** [explosion/spaCy#6122] [Language Support]
I have reviewed the list of [Tagalog stop words](https://github.com/explosion/spaCy/blob/master/spacy/lang/tl/stop_words.py), and I thought the following tokens could be added:

```
{'ba',
 'eh',
 'kasi',
 'lang',
 'mo',
 'naman',
 'opo',
 'po',
 'si',
 'talaga',
 'yung'}
```

Thank you.

**Comment by svlandeg** [comment]
If these are truely generic stop words like "the" in English, I think it's fine to create a PR and merge them in the current list. However do consider that the definition of a "stop word" is somewhat vague. What is meaningful for one text/application, may not be meaningful for another and vice versa, and I would prefer to err on the side of having too few stop words rather than too many. 

That said, you will be better positioned to judge the "stop word-i-ness" of these words than I would ever be, as I don't speak the language, so we'd welcome your insights & contribution!
